scripts/Dockerfile

```diff
@@ -50,11 +56,14 @@ COPY ./scripts/VERSION .
 
 # -ldflags="-w -s" means omit DWARF symbol table and the symbol table and debug information
 # go-sqlite3 requires CGO_ENABLED
-RUN VERSION=`cat ./VERSION`${VERSION_SUFFIX} && CGO_ENABLED=1 GOOS=linux GOARCH=amd64 go build \
-    --tags "${RELEASE},embed_frontend" \
-    -ldflags="-w -s -X 'github.com/bytebase/bytebase/backend/bin/server/cmd.version=${VERSION}' -X 'github.com/bytebase/bytebase/backend/bin/server/cmd.goversion=${GO_VERSION}' -X 'github.com/bytebase/bytebase/backend/bin/server/cmd.gitcommit=${GIT_COMMIT}' -X 'github.com/bytebase/bytebase/backend/bin/server/cmd.buildtime=${BUILD_TIME}' -X 'github.com/bytebase/bytebase/backend/bin/server/cmd.builduser=${BUILD_USER}'" \
-    -o bytebase \
-    ./backend/bin/server/main.go
+RUN if [ "$TARGETARCH" = "arm64" ]; then \
```

Comment:
Can we use "if else fi" style here too?

Reply:
> Can we use "if else fi" style here too?

Yes, If in "amd64" we can set `CC` to `gcc`. However, I apologize that I may not have enough time now to build another image and test it. The current code has been tested by me and I've built images for arm64 and amd64 in my private repository. You can see it in https://hub.docker.com/repository/docker/941102297/bytebase/tags?page=1&ordering=last_updated.